pkg/graph/resolvers/resolver_test.go

```diff
@@ -390,6 +390,24 @@ func (suite *ResolverSuite) createDefaultTestAnalyzedAudit(mp *models.ModelPlan,
 
 }
 
+func (suite *ResolverSuite) createMilestoneCommon(
```

Comment:
@OddTomBrooks, this is fine here, wanted to see what you thought about moving to the mto_milestone test file instead though? I think it might feel a bit better organizationally, though we aren't doing this now.

Reply:
I think this is worth a higher level discussion on how we want to organize the logic in our test suite, let's have a chat after standup today